man/systemd.link.xml

```diff
@@ -443,6 +443,79 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>Advertising=</varname></term>
+        <listitem>
+          <para>The Advertising option is used to set what speeds and modes of operation
+          (duplex mode) are advertised for auto-negotiation. The
+          supported values are:
+
+          <table>
+            <title>Supported advertise values</title>
+            <tgroup cols='5'>
+             <colspec colname='Advertise' />
+             <colspec colname='Hex Value' />
```

Comment:
i don't think we should ever show the hex value to anyone. that's low-level nerd stuff, and shouldn't be visible in the docs or in any friendly UI, please drop this

Reply:
OK